packages/cli/package.json

```diff
@@ -17,15 +17,14 @@
     },
     "license": "SEE LICENSE IN LICENSE FILE IN GIT REPOSITORY",
     "scripts": {
-        "postbuild": "npm run copy:types",
         "copy:files": "copyfiles -u 1 lib/templates/* dist",
-        "copy:types": "cp `node -p \"require.resolve('@nangohq/shared/dist/sdk/sync.d.ts')\"` './dist/nango-sync.d.ts'",
```

Comment:
this one is a big bet, I'm trying to load the file from user's global node_modules, should work but let's see :D